modules/apps/style-book/style-book-web-test/src/testIntegration/java/com/liferay/style/book/web/internal/portlet/action/test/ExportImportStyleBookEntriesMVCResourceCommandTest.java

```diff
@@ -213,11 +220,15 @@ public void testExportImportSingleStyleBookEntryAndOverwrite()
 			ServiceContextTestUtil.getServiceContext(
 				_sourceGroup, TestPropsValues.getUserId());
 
+		String name = RandomTestUtil.randomString();
```

Comment:
You should declare the variable closer to its use.
I see that `name` is used when you create the variable `StyleBookEntry updatedStyleBookEntry =`. Can you please declare `name` closer to its usage?